chart/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
 name: obot
 description: A Helm chart for Obot
-version: 0.1.0
+version: 0.1.1
```

Comment:
no need to update this, the version is set dynamically when we do a release of obot.